Client/Frontend/Home/DownloadsPanel.swift

```diff
@@ -305,7 +402,6 @@ class DownloadsPanel: UIViewController, UITableViewDelegate, UITableViewDataSour
         let delete = UITableViewRowAction(style: .destructive, title: deleteTitle, handler: { (action, indexPath) in
             if let downloadedFile = self.downloadedFileForIndexPath(indexPath) {
                 if self.deleteDownloadedFile(downloadedFile) {
```

Comment:
To be safe. (I cant find this in the documentation right now) But you want to do the actual deletion inside the animation block.

Reply:
`If you do not make the insertion, deletion, and selection calls inside this block, table attributes such as row count might become invalid. `
https://developer.apple.com/documentation/uikit/uitableview/1614908-beginupdates